packages/rfw/lib/src/dart/binary.dart

```diff
@@ -468,6 +474,14 @@ class _BlobDecoder {
     return ConstructorCall(name, _readMap(_readArgument)!);
   }
 
+  WidgetBuilderDeclaration _readWidgetBuilder() {
+    final String argumentName = _readString();
+    final int type = _readByte();
+    assert(type == _msWidget || type == _msSwitch);
```

Comment:
what guarantees that this type is only one of those two values?

i would have expected this to be a FormatException rather than a debug-only assert.